generated/models/spend_tx.go

```diff
@@ -20,14 +20,14 @@ type SpendTx struct {
 	// amount
 	// Required: true
 	// Minimum: 0
-	Amount *int64 `json:"amount"`
```

Comment:
shouldn't this be big.Int ?

Reply:
true, I only converted Account Balance to bigint